sqs/main.tf

```diff
@@ -80,3 +79,19 @@ module "dlq_cloudwatch_alarm" {
   }
   notification_topic = var.dlq_notification_topic
 }
+
+module "queue_cloudwatch_alarm" {
+  source              = "../cloudwatch_alarms"
+  metric_name         = "ApproximateNumberOfMessagesVisible"
+  namespace           = "AWS/SQS"
+  name                = "${var.queue_name}-alarm"
+  threshold           = var.queue_cloudwatch_alarm_visible_messages_threshold
+  comparison_operator = "GreaterThanThreshold"
+  statistic           = "Sum"
+  treat_missing_data  = "ignore"
+  datapoints_to_alarm = 1
+  dimensions = {
+    QueueName = local.sqs_queue.name
+  }
+  notification_topic = var.dlq_notification_topic
```

Comment:
Is `var.dlq_notification_topic` correct?

Reply:
Ah no, it needs a new variable. I've added on in.